docs/resources/user_attribute_group.md

```diff
@@ -0,0 +1,66 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "looker_user_attribute_group Resource - terraform-provider-looker"
+subcategory: ""
+description: |-
+  Manages Looker User Attribute Groups
+---
+
+# looker_user_attribute_group (Resource)
+
+Manages Looker User Attribute Groups
+
+## Example Usage
+
+```terraform
+  resource "looker_group" "test_group_1" {
+    name = "Test_1"
+  }
+
+  resource "looker_group" "test_group_2" {
+    name = "Test_2"
+  }
+
+  resource "looker_user_attribute" "test_attr" {
+    name          = "test_name"
+    label         = "test_label"
+    data_type     = "number"
+    hidden        = false
+    default_value = 123
+    user_access   = "View"
+  }
+
+  resource "looker_user_attribute_group" "test_group_1" {
+    user_attribute_id = looker_user_attribute.test_attr.id
+    group_values {
+      group_id = looker_group.test_group_1.id
+      value    = "0"
+    }
+    group_values {
+      group_id = looker_group.test_group_2.id
+      value    = "1"
+    }
+  }
+```
+
+<!-- schema generated by tfplugindocs -->
+
+## Schema
+
+### Required
+
+- `group_values` (Block List, Min: 1) (see [below for nested schema](#nestedblock--group_values))
+- `user_attribute_id` (String) The id of the user attribute to assign
+
+### Read-Only
+
+- `id` (String) The unique id of the user attribute group
+
+<a id="nestedblock--group_values"></a>
+
+### Nested Schema for `group_values`
+
+Required:
+
+- `group_id` (String) The id of the group to set user attribute for
+- `value` (String) Value of attribute overriding any existing default value
```

Comment:
It may be worth documenting how to do an import in the docs here? I've been doing it manually but I'm working on improving the generator to do it for us :)

Reply:
Good idea, I'll just add it manually for now